priv/catalog/845CC9.yaml

```diff
@@ -0,0 +1,56 @@
+id: 845CC9
+name: Corosync configuration file
+group: Corosync
+description: |
+  Corosync `max_messages` is set to expected value
+remediation: |
+  ## Abstract
+
+  ## Remediation
+
+  Adjust the corosync `max_messages` count as recommended on the best practices, and reload the corosync configuration
+
+  1. Set the correct `max_messages` count in the `totem` section in the corosync configuration file `/etc/corosync/corosync.conf`. This action must be repeated in all nodes of the cluster.
+     ```
+     [...]
+     totem {
+            max_messages: <count value>
+           }
+     [...]
+     ```
+  2. Reload the corosync configuration:
+     `crm corosync reload`
+
+  ## References
+  Azure:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker#install-the-cluster
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-example-for-etccorosynccorosync-conf
+
+  GCP: 
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles
+
+  AWS:
+
+    -	https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-example-for-etccorosynccorosync-conf
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-example-for-etccorosynccorosync-conf
+
+facts:
+  - name: corosync_max_messages
+    gatherer: corosync.conf
+    argument: totem.max_messages
+
+values:
+  - name: expected_max_messages
+    default: 20
+
+expectations:
+  - name: retransmits
```

Comment:
I didn't get your point. Check 21FCA6 is about token_retransmits_before_loss_const and this check is about max_messages. Both of them are **not** timeout values.

Reply:
Maybe I wasn't clear enough.

What I meant is that this `retransmits` expectation name in the current check `845CC9` sounds inappropriate to me because, as you say, current check `845CC9` is about `max_messages`.

So I am proposing to change this `retransmits` (which is appropriate in `21FCA6`) to something related to `845CC9` and `max_messages`.

Does this clarify a bit?

Reply:
Now I got your point and you are right. Change is committed.